.github/workflows/node.js.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js
+
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+      #    - name: Use Node.js ${{ matrix.node-version }}
+      #      uses: actions/setup-node@v1
+      #      with:
+      #        node-version: ${{ matrix.node-version }}
+
+    - run: npm install
+      working-directory: client
+
+    - run: npm test
```

Comment:
Are we doing an npm test in other workflows at this point? or is the first one?

Can we tie it into another workflow?